tiledb/ml/__init__.py

```diff
@@ -0,0 +1,11 @@
+from pkg_resources import DistributionNotFound, get_distribution  # type: ignore
```

Comment:
The `# type: ignore` can be avoided here if we install the missing mypy stub packages with `mypy --install-types  --non-interactive` on CI.

Reply:
Yup. It didn't work locally in my case. I can check it again, though.